docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,123 @@
+.. _intro/get_started:
+
+****************
+Getting Started
+****************
+
+Installation
```

Comment:
```suggestion
Install with conda
```
And that should then be on its own page.

Reply:
I'm not clear what you are expecting to remain on this page. If we move the conda install to a separate page, then surely there will be nothing left on this page?

Reply:
The panels for the main installation routes is left.